File: Algorithm/leetcode_001.go
/**
 * File              : leetcode_001.go
 * Author            : Andy
 * Date              : 2022.08.08
 * Last Modified Date: 2022.08.08
 * Last Modified By  : Andy
 */
;

func twoSum(nums []int, target int) []int{
	for i, x := range nums{
		for j:= i+1;j < len(nums); j++{
			if x + nums[j] == target{
				return []int{i, j}
			}
		}
	}
	return nil
}

File: Algorithm/leetcode_001.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# File              : leetcode_001.py
# Author            : Andy
# Date              : 2022.08.08
# Last Modified Date: 2022.08.08
# Last Modified By  : Andy


class Solution:
    def twoSum(self, nums, target):
        for index, val in enumerate(nums):
            res = target - val
            if res in nums and nums.index(res) != index:
                return index, nums.index(res)
        return -1, -1

